torch_em/segmentation.py

```diff
@@ -316,8 +316,11 @@ def default_segmentation_dataset(
     return ds
 
 
-def get_data_loader(dataset: torch.utils.data.Dataset, batch_size, **loader_kwargs) -> torch.utils.data.DataLoader:
-    loader = torch.utils.data.DataLoader(dataset, batch_size=batch_size, **loader_kwargs)
+def get_data_loader(dataset: torch.utils.data.Dataset, batch_size: int, **loader_kwargs) -> torch.utils.data.DataLoader:
+    pin_memory = loader_kwargs.pop("pin_memory", True)
+    print(pin_memory)
+    breakpoint()
```

Comment:
It seems like you missed some debugging output ;)

Reply:
Ah yes, sorry! I removed it in the latest commit. Thanks for spotting!